client/my-sites/stats/stats-list/action-spam.jsx

```diff
@@ -37,47 +37,50 @@ class StatsActionSpam extends React.Component {
 			this.props.afterChange( ! this.state.spammed );
 		}
 
-		wpcomSite = wpcom.site( this.props.data.siteID );
+		const wpcomSite = wpcom.site( this.props.data.siteID );
 		wpcomSite[ spamType ].call( wpcomSite, this.props.data.domain, function() {} );
 		analytics.ga.recordEvent( 'Stats', gaEvent + ' in ' + this.props.moduleName + ' List' );
 	};
 
 	render() {
-		let label = this.state.spammed
-				? this.props.translate( 'Not Spam' )
-				: this.props.translate( 'Spam', {
-						context: 'Stats: Action to mark an item as spam',
-						comment: 'Default label (changes into "Not Spam").',
-				  } ),
-			title = this.state.spammed
-				? this.props.translate( 'Not Spam', {
-						textOnly: true,
-						context: 'Stats: Action to undo marking an item as spam',
-						comment:
-							'Secondary label (default label is "Spam"). Recommended to use a very short label.',
-				  } )
-				: this.props.translate( 'Spam', {
-						textOnly: true,
-						context: 'Stats: Action to mark an item as spam',
-						comment: 'Default label (changes into "Not Spam").',
-				  } ),
-			wrapperClass = classNames( 'module-content-list-item-action-wrapper', {
-				spam: ! this.state.spammed,
-				unspam: this.state.spammed,
-			} );
+		const label = this.state.spammed
+			? this.props.translate( 'Mark as Not Spam' )
+			: this.props.translate( 'Mark as Spam', {
+					context: 'Stats: Action to mark an item as spam',
+					comment: 'Default label (changes into "Mark as Not Spam").',
+			  } );
+		const title = this.state.spammed
+			? this.props.translate( 'Mark as Not Spam', {
+					textOnly: true,
+					context: 'Stats: Action to undo marking an item as spam',
+					comment:
+						'Secondary label (default label is "Spam"). Recommended to use a very short label.',
```

Comment:
Shall we update this translator comment? The default label is no longer 'Spam'.

Reply:
Yes, I don't think they're necessary anymore though so I just removed them :)